### PR TITLE
Feature: update LibRI v0.2.0

### PR DESCRIPTION
**Pull request by PeizeLin**
LibRI has been updated to v0.2.0, whose interface is not backward compatible.
This is the update to the calling interface.

### CODE REVIEW

File: source/module_io/read_input_item_exx_dftu.cpp [hunk 1]
```diff
@@ -149,6 +149,18 @@ void ReadInput::item_exx()
         read_sync_double(input.exx_v_grad_threshold);
         this->add_item(item);
     }
+    {
+        Input_Item item("exx_c_grad_r_threshold");
```
Comment (mohanchen) [inline source/module_io/read_input_item_exx_dftu.cpp:153]: could you change 'threshold' to 'thr' to be consistent with other vaiables?

Comment (mohanchen) [reply source/module_io/read_input_item_exx_dftu.cpp:153]: another issue is the input variable name may be a bit too long, is it possible to be shorter?

**Comment by QuantumMisaka** [comment]
@PeizeLin  Any Update ?
Toolchain 2024.2 is ongoing, I'm looking forward to using the newest LibRI

**Comment by maki49** [comment]
I think this PR is ready to merge.

**Review by mohanchen — APPROVED** [review]

**Pull request merged**